### ISSUE TITLE
injecting reducers isn't working with aot [workaround]

### ISSUE THREAD

**Issue by Szpadel** [ngrx/platform#386]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Feature request
[X] Documentation issue or request
</code></pre>

## What is the current behavior?
When doing reducer injection as described [here](https://github.com/ngrx/platform/blob/master/docs/store/api.md#injecting-reducers) when using aot, InjectionToken is injected as reducer and that causes then chain of errors resulting reducer to be undefined and addFeature is crashing.

## Expected behavior:
This seams to be angular issue, this issue might be helpful for others to understand what is happening.

## Other information:
I found workaround for that issue (you might want to put it in documentation)
you have to use `{provide: ..., useFactory: ...}` instead of `{provide: ..., useValue: ...}`
eg.:
```
const featureName = '...';
export const FEATURE_REDUCER = new InjectionToken<ActionReducer<...>>(featureName);

@NgModule({
  imports: [
    StoreModule.forFeature(featureName, FEATURE_REDUCER, {initialState: defaultState})
  ],
  providers: [
    {provide: FEATURE_REDUCER, useFactory: createReducer }
  ]
})
```
Also make sure that you have `export` for InjectionToken, because there is another angular issue that causes NgModule class from the same file to be used instead of InjectionToken when it isn't exported.

**Comment by brandonroberts** [comment]
Thanks! This is just an error in the docs as both examples should be using a factory function. Mind making a PR to fix it?

**Comment by brandonroberts** [comment]
Fixed via #387
